src/Music/Prelude/Standard.hs

```diff
@@ -65,55 +66,70 @@ import Music.Score.Export.StandardNotation (Asp1, Asp1a, LilypondLayout (..), Li
 import qualified Music.Score.Part
 import qualified System.Environment
 import qualified Text.Pretty
+import qualified Options.Applicative as O
 
 type StandardNote = Asp1
 
 type Music = Score StandardNote
 
 -- TODO set logging level
 -- TODO more options?
-data CommandLineOptions
-  = ToXml FilePath
-  | ToLy LilypondOptions FilePath
-  | ToMidi FilePath
-  | ToLyAndMidi LilypondOptions FilePath
-  | Help
-
--- TOOD Proper parser using optparse-applicative or optparse-generic
-parse :: Applicative m => [String] -> m CommandLineOptions
-parse ("-f" : "xml" : "-o" : path : _) = pure $ ToXml path
-parse ("-f" : "ly" : "-o" : path : _) = pure $ ToLy defaultLilypondOptions path
-parse ("-f" : "ly+mid" : "-o" : path : _) = pure $ ToLyAndMidi defaultLilypondOptions path
-parse ("-f" : "ly" : "--layout=inline" : "-o" : path : _) = pure $ ToLy (defaultLilypondOptions {layout = LilypondInline}) path
-parse ("-f" : "mid" : "-o" : path : _) = pure $ ToMidi path
-parse _ = pure Help
+data ExportBackend = ToXml | ToLy | ToMidi | ToLyAndMidi
+data CommandLineOptions = CommandLineOptions ExportBackend LilypondOptions FilePath
```

Comment:
```suggestion
data CommandLineOptions = CommandLineOptions 
  { backend :: ExportBackend
  -- ^ Backend to use.
  , lilypondOptuions :: LilypondOptions
  -- ^ Lilypond options, ignored when `backend` is not `ToLy`.
  , output :: FilePath
  -- ^ Path to generated file.
  }
```

Reply:
Changed and refactored to use the record selectors

Reply:
Looks, great, merging!